sixtyfps_compiler/parser/expressions.rs

```diff
@@ -85,12 +87,27 @@ fn parse_expression_helper(p: &mut impl Parser, precedence: OperatorPrecedence)
         }
     }
 
-    if p.nth(0).kind() == SyntaxKind::LParent {
-        {
-            let _ = p.start_node_at(checkpoint.clone(), SyntaxKind::Expression);
+    loop {
+        match p.nth(0).kind() {
```

Comment:
Is this the match where the `SyntaxKind::LBracket` will go for `IndexExpression` later?

Reply:
yes, exactly.
Yhis will allow the parser to understand things like `foo()[34][22].xx`  for example